src/k8s/pkg/utils/k8s/restart_deployment.go

```diff
@@ -0,0 +1,27 @@
+package k8s
+
+import (
+	"context"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// RestartDeployment updates the restartedAt field to trigger a rollout restart for the given Deployment.
+func RestartDeployment(ctx context.Context, client *k8sClient, name, namespace string) error {
+	if namespace == "" {
+		namespace = "default"
+	}
+	deployment, err := client.AppsV1().Deployments(namespace).Get(ctx, name, metav1.GetOptions{})
+	if err != nil {
+		return err
```

Comment:
let's wrap the error here, to add more context for the error messages that surface